builtin/logical/database/backend.go

```diff
@@ -135,7 +135,7 @@ func (b *databaseBackend) DatabaseConfig(ctx context.Context, s logical.Storage,
 type upgradeStatements struct {
 	// This json tag has a typo in it, the new version does not. This
 	// necessitates this upgrade logic.
-	CreationStatements   string `json:"creation_statments"`
+	CreationStatements   string `json:"creation_statements"`
```

Comment:
[For others in the Vault team] This may not be required.

Reply:
In fact, this would break things because this is on purpose. The real question is, since this commit is automated, will the author respond to comments.

Reply:
There's a sleepy human behind this. I'll drop this